docs/Contribute-to-AlmaLinux-Build-System.md

```diff
@@ -0,0 +1,187 @@
+  ```
+  or
+  ```
+  Resolves: AlmaLinux/build-system/issues/80
+  ```
+  It's optional, and can be used only when the commit message is short enough to fit in 72 characters.
+
+#### Body
+
+Body is a more detailed explanatory text. Wrap it to 72 characters. The blank line separating the summary from the body is critical (unless you omit the body entirely).
+
+Further paragraphs come after blank lines.
+
+* Bullet points are okay, too.
+* Typically a hyphen or asterisk is used for the bullet, followed by a single space. Use a hanging indent.
+
+When writing the body part, you can add as much details as you want on the commit message. Just remember:
+* The body is optional if the subject is enough to describe the commit.
+* Wrap lines to 72 characters.
+* Using the imperative or simple past tenses is acceptable, but please be consistent.
+* Feel free to describe what you think it's worth being described as if you were your colleague (that is more or less familiar to the code) that would like to understand what happened by just reading a few lines.
+
+#### Footer
+
+Footer is a perfect place to add references as links to fixed or resolved issue(s). If have not referenced issues in the subject, then they can be added here:
+```
+Fixes https://github.com/AlmaLinux/build-system/issues/80
+```
+or 
+```
+Resolves https://github.com/AlmaLinux/build-system/issues/80
+```
+
+Remember:
+* Not having referenced issues is a possibility in case of dependency version updates and so on.
+* Can use a bullet list for the list of references.
+* Although using Github's abbreviations is okay, we privilege using the full URL as it will be meaningful if somebody is not reading the commit message through Github's interface.
+
+### Everyday Use cases
+
+#### The simplest commit message
+
+```
+Fixed create_module_by_payload (#123)
+```
+As you can see, simple commit messages like this one are ok. Even if not providing a good description in the commit message, it:
+
+* Describes that there was a fix in `create_module_by_payload`.
+* References a github issue (#123).
+
+However, this structure might not fit as `#123` will refer to issues/pull request within the same repository. That reference needs to be more specific.
+
+#### A tad more complex one
+
+```
+Added some missing AlmaLinux beta repositories
+
+We are adding missing AlmaLinux 8 RT repos for s390x and ppc64le
+Also, adding aarch64 NFV repos for AlmaLinux 9
+
+Resolves: https://github.com/AlmaLinux/build-system/issues/80
+```
+
+#### A little bit about merge commits
+
+Some commits might not be meaningul, others might not include references, some can ollow the same structure or are not fully descriptive. Since 99% of the time we merge pull requests (hereinafter PRs), we should add the relevant data in the mere commit message. By relevant data we understand the branch the code is coming from and the information we would like to know about the commits as described earlier in the document.
```

Comment:
```suggestion
Some commits might not be meaningful or might not include references to specific GitHub issues, so they cannot follow the same structure or be as fully descriptive. Since 99% of the time we merge pull requests (hereinafter PRs), we should add the relevant data in the commit message. By relevant data we mean the branch the code is coming from, and the information we would like to know about the commits as described earlier in the document, so that it can be understood later.
```
I'm not entirely sure I understood what this paragraph was trying to say, so if my suggestions weren't good, please feel free to say so.